scripts/us_bea/states_gdp/us_state_gdp_quarterly_import_config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+  import_name: "USStatesQuarterlyGDP"
+  provenance_url: "https://www.bea.gov/data/gdp/gdp-state"
+  provenance_description: "Quarterly data on inflation-adjusted GDP at a state level in the United States"
```

Comment:
Please include the source in the description "... from the US Bureau of Economic Analysis

Reply:
Done.